content/en/users/datatransfer/WebFTS/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "WebFTS"
+linkTitle: "WebFTS"
+type: docs
+description: "Documentation related to EGI Data Transfer Web Interface"
+weight: 30
+---
+
+## Overview 
+
+The WebFTS interface offers end users a way to graphically execute and monitor file transfer between storages using different protocols.
+
+The full documentation is accessible at:
+
+https://fts3-docs.web.cern.ch/fts3-docs/docs/webfts/userguide.html
+
+with FAQ at:
+
+https://fts3-docs.web.cern.ch/fts3-docs/docs/webfts/faq.html
+
+
+### Credential Delegation
+
+The authentication with the  Storage Elements (SE) is done based on X509 certificate which needs to be installed on the browser. The Integration with EGI Check-in is under development and will be available soon in the production instance.
+
+A pop-up window appears when you choose the "My jobs" or "Submit a transfer" tabs. There, you will need to paste the private RSA key of your certificate. The private key WILL NOT BE TRANSMITTED ANYWHERE. It is only used locally (within the user's browser) to generate the proxies needed to have access to the FTS services.
+
+![image](webfts_delegate.png)
+
+To obtain the private key, you can run in the console:
+
+``` {.console}
+openssl pkcs12 -in yourCert.p12 -nocerts -nodes | openssl rsa
+```
+
+This command requires openssl.
+
+If you need a delegation with VOMS credentials (required to access some types of SE endpoints), you will need to introduce the name of the virtual organization (VO) you belong to. 
+
+From the right button where the remaining time for the current delegation is shown, you can remove the current delegation and delegate again (the delegation window will appear).
+
+### Submitting transfers
+
+Open the "Submit a transfer" tab and Load origin and destination storage elements as endpoints.
```

Comment:
You may wan to use `Submit a transfer` or **Submit a transfer** or _Submit a transfer_ to represent buttons/texts